--- a/man/dracut.8.asc
+++ b/man/dracut.8.asc
@@ -24,7 +24,7 @@
 _/boot/<machine-id>/<kernel-version>/initrd_,
 _/boot/efi/<machine-id>/<kernel-version>/initrd_,
 _/lib/modules/<kernel-version>/initrd_ or
-_/boot/initramfs-<kernel-version>.img_.
+_/boot/initrd.img-<kernel-version>_.
 
 dracut creates an initial image used by the kernel for preloading the block
 device modules (such as IDE, SCSI or RAID) which are needed to access the root
--- a/dracut.sh
+++ b/dracut.sh
@@ -1140,7 +1140,7 @@
         elif [[ -f "$dracutsysrootdir"/lib/modules/${kernel}/initrd ]]; then
             outfile="$dracutsysrootdir/lib/modules/${kernel}/initrd"
         elif [[ -e $dracutsysrootdir/boot/vmlinuz-${kernel} ]]; then
-            outfile="$dracutsysrootdir/boot/initramfs-${kernel}.img"
+            outfile="$dracutsysrootdir/boot/initrd-${kernel}"
         elif [[ -z $dracutsysrootdir ]] \
             && [[ $MACHINE_ID ]] \
             && mountpoint -q /efi; then
@@ -1150,7 +1150,7 @@
             && mountpoint -q /boot/efi; then
             outfile="/boot/efi/${MACHINE_ID}/${kernel}/initrd"
         else
-            outfile="$dracutsysrootdir/boot/initramfs-${kernel}.img"
+            outfile="$dracutsysrootdir/boot/initrd-${kernel}"
         fi
     fi
 fi
--- a/man/dracut.usage.asc
+++ b/man/dracut.usage.asc
@@ -10,7 +10,7 @@
 _/boot/_++<machine-id>++_/_++<kernel-version>++_/initrd_,
 _/boot/efi/_++<machine-id>++_/_++<kernel-version>++_/initrd_,
 _/lib/modules/_++<kernel-version>++_/initrd_ or
-_/boot/initramfs-_++<kernel-version>++_.img_ and contains the kernel modules of
+_/boot/initrd-_++<kernel-version>++_ and contains the kernel modules of
 the currently active kernel with version _++<kernel-version>++_.
 
 If the initramfs image already exists, dracut will display an error message, and
--- a/man/lsinitrd.1.asc
+++ b/man/lsinitrd.1.asc
@@ -22,7 +22,7 @@
 _/boot/<machine-id>/<kernel-version>/initrd_,
 _/boot/efi/<machine-id>/<kernel-version>/initrd_,
 _/lib/modules/<kernel-version>/initrd_ or
-_/boot/initramfs-<kernel-version>.img_.
+_/boot/initrd-<kernel-version>_.
 
 OPTIONS
 -------
--- a/lsinitrd.sh
+++ b/lsinitrd.sh
@@ -131,8 +131,8 @@
         image="/boot/efi/${MACHINE_ID}/${KERNEL_VERSION}/initrd"
     elif [[ -f /lib/modules/${KERNEL_VERSION}/initrd ]]; then
         image="/lib/modules/${KERNEL_VERSION}/initrd"
-    elif [[ -f /boot/initramfs-${KERNEL_VERSION}.img ]]; then
-        image="/boot/initramfs-${KERNEL_VERSION}.img"
+    elif [[ -f /boot/initrd-${KERNEL_VERSION} ]]; then
+        image="/boot/initrd-${KERNEL_VERSION}"
     elif [[ $MACHINE_ID ]] \
         && mountpoint -q /efi; then
         image="/efi/${MACHINE_ID}/${KERNEL_VERSION}/initrd"
